google-native-csharp/${PROJECT}.csproj

```
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Pulumi.GoogleNative" Version="0.0.*-alpha*" />
```

```suggestion
    <PackageReference Include="Pulumi.GoogleNative" Version="0.*" />
```
I assume we'll publish 0.1.0 before merging

---

Done.